File: Modul-05/SOAL 4/PRAK504-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int REVERSE(int num) {
    int REVERSED = 0;
    while (num > 0) {
        REVERSED = REVERSED * 10 + num % 10;
        num /= 10;
    }
    return REVERSED;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);

    A = REVERSE(A);
    B = REVERSE(B);

    int C = A + B;
    printf("%d", REVERSE(C));

    return 0;
}

File: Modul-05/SOAL 4/PRAK504-2310817210009-MuhammadIbnuSina.py
def REVERSE(num):
    REVERSED_num = 0
    while num > 0:
        REVERSED_num = REVERSED_num * 10 + num % 10
        num //= 10
    return REVERSED_num

def main():
    A, B = map(int, input().split())

    A = REVERSE(A)
    B = REVERSE(B)

    C = A + B
    print(REVERSE(C))

if __name__ == "__main__":
    main()
